tests/core/test_abort_on_exception.cpp

```diff
@@ -27,5 +27,9 @@ EMSCRIPTEN_BINDINGS(test_abort_on_exception) {
 }
 
 int main() {
+#if CRASH_IN_MAIN
```

Comment:
I don't see this used in the test runner code?

Reply:
Oops, part of an earlier version, reverted.